src/ansible_creator/utils.py

```diff
@@ -337,7 +334,7 @@ def _per_container(self, resource: str, current_index: int) -> FileList:
                     data=template_data,
                 )
                 deserialized = yaml.safe_load(templated)
-                setattr(template_data, key, deserialized)
+                setattr(template_data, key, deserialized)  # Use the deserialized templated value
```

Comment:
If this is necessary can it go above the line rather than appended too it? That is the more common pattern in the code base